pyproject.toml

```diff
@@ -66,7 +66,7 @@ pandas = ">=1.2.0"
 numpy = ">=1.20.0"
 tox = "^3.23.1"
 freezegun = "^1.1.0"
-viztracer = "^0.14.0"
+viztracer = "^0.15.2"
```

Comment:
GitLab's runners were having trouble finding the wheel for the older version, so figured I'd just bump it.